### ISSUE TITLE
Cannot build on MacOSX using XCode.

### ISSUE THREAD

**Issue by bluenex** [pwsafe/pwsafe#438]
## System
- MacOSX - `10.13.4`
- XCode - `9.3`

## Details
I did all steps provided in [README.MAC.DEVELOPERS.txt](https://github.com/pwsafe/pwsafe/blob/master/README.MAC.DEVELOPERS.txt), and I believe that this error does not relate to other steps.

The error text and image are listed below:

```
- Semantic Issue Group
  - /Users/***/pwsafe/src/os/mac/KeySend.cpp:27:21: Non-const lvalue reference to type 'const unsigned char *' cannot bind to a value of unrelated type 'unsigned char *'
    - /Users/***/pwsafe/src/core/UTF8Conv.h:30:58: Passing argument to parameter 'utf8' here
```

![image](https://user-images.githubusercontent.com/11027706/38292424-e3f5a4d4-380d-11e8-8f7e-c96613f7f8f2.png)

**Comment by ronys** [comment]
I don't have a Mac development environment, but I think if you change line 25 of src/os/mac/KeySend.cpp from
unsigned char* str = 0;
to
const unsigned char* str = 0;

then all should compile cleanly.

If you can confirm this works, I'll commit the change.

**Comment by ronys** [comment]
Fixed in commit 50af626e7
